src/pymoca/ast.py

```diff
@@ -631,8 +692,13 @@ def _find_class(self, component_ref: ComponentRef, search_parent=True, search_im
                 else:
                     raise ClassNotFoundError("Could not find class '{}'".format(component_ref))
 
-    def find_class(self, component_ref: ComponentRef, copy=True,
-                   check_builtin_classes=False, search_imports=True) -> 'Class':
+    def find_class(
```

Comment:
I guess the black format highlights ugly functions that maybe should be cleaned up haha!